src/account/operations/address_generation.rs

```diff
@@ -93,10 +82,12 @@ impl AccountHandle {
                 // If we don't sync, then we want to display the prompt on the ledger with the address. But the user
                 // needs to have it visible on the computer first, so we need to generate it without the
                 // prompt first
-                if !options.metadata.ledger_nano_prompt {
-                    let mut changed_metadata = options.metadata.clone();
-                    // Change metadata so ledger will not show the prompt the first time
-                    changed_metadata.ledger_nano_prompt = true;
+                if options.options.clone().unwrap_or_default().ledger_nano_prompt {
```

Comment:
No more `!` ?

Reply:
Yes, currently it's wrong :(
It was initially there for `!syncing`, where true meant, that it will not show it on the ledger. Then we renamed the syncing field to ledger_nano_prompt and in iota.rs what it does, but in wallet.rs we just renamed it without changing this

Reply:
Ok I'm not fully sure to understand your answer but if you think this is correct, all good

Reply:
The code on develop was updated in the meantime already :trollface: